doc/designs/subordinate-ids.md

```diff
@@ -64,6 +64,13 @@ and don't auto-map or auto-assign subordinate ids by default. Instead
 we give the admin several options to assign them manually, semi-manual,
 or automatically.
 
+For deployments where there is a need to consume IDs above 2^31 for normal UID
+and GID assignments, one has to disable subordinate ID usage. This should be
+done with `ipa config-mod --addattr ipaconfigstring=SubID:Disable` command.
+After it is done, subordinate ID range can be removed with `ipa idrange-del`
```

Comment:
bike-shedding: is "capability" or "feature" better than usage? usage to me implies it is in use. If it's in-use then it can't be disabled.

Reply:
'feature' is better, I think. I'll change it.